docs/source/distributions/kubernetes_deployment.md

```diff
@@ -47,28 +52,23 @@ spec:
         app.kubernetes.io/name: vllm
     spec:
       containers:
-      - name: llama-stack
-        image: $(VLLM_IMAGE)
-        command:
-            - bash
-            - -c
-            - |
-              MODEL="meta-llama/Llama-3.2-1B-Instruct"
-              MODEL_PATH=/app/model/$(basename $MODEL)
-              huggingface-cli login --token $HUGGING_FACE_HUB_TOKEN
-              huggingface-cli download $MODEL --local-dir $MODEL_PATH --cache-dir $MODEL_PATH
```

Comment:
Does that mean vllm itself download the model?

Reply:
Yes. See https://docs.vllm.ai/en/stable/design/huggingface_integration.html